optaplanner-website-root/content/code/continuousIntegration.html

```diff
@@ -0,0 +1,4 @@
+title=Continuous integration
+type=redirectBase
+redirect_url=https://www.optaplanner.org/code/sourceCode.html
+~~~~~~
```

Comment:
I am using the *.html extension so it easier to see the "real" files in the sources, as those have the *.adoc extension.